### PR TITLE
[JENKINS-72148] Add missing fr properties in package jenkins

### PR DESCRIPTION
**Pull request by jgreffe**
See [JENKINS-72148](https://issues.jenkins.io/browse/JENKINS-72148).
Add and translate all the missing *_fr.properties in src/main/resources/jenkins.

### Proposed changelog entries

- JENKINS-72148, add missing fr properties in jenkins resources

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [x] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@mention

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

**Comment by lemeurherve** [comment]
Reviewing it.

File: core/src/main/resources/jenkins/tasks/filters/impl/RetainVariablesLocalRule/config_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/jenkins/tasks/filters/impl/RetainVariablesLocalRule/config_fr.properties:24]:
```suggestion
  Les variables d''environnement définies hors de Jenkins seront soit réinitialisées à leur valeur par défaut soit supprimées en fonction de l''option "Gestion des variables d''environnement", sauf si spécifiées ici.
```

File: core/src/main/resources/jenkins/tasks/filters/impl/RetainVariablesLocalRule/config_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/jenkins/tasks/filters/impl/RetainVariablesLocalRule/config_fr.properties:25]:
```suggestion
retainCharacteristicEnvVarsLabel=Garde les variables d''environnement spécifiques
```
Make more sense in French like that IMO (optional)

**Review by lemeurherve — COMMENTED** [review]
Another big PR! 🤯 

Note: my `NB:` comments concern original messages in English, they don't have to be taken in account in this PR.

File: core/src/main/resources/jenkins/monitor/OperatingSystemEndOfLifeAdminMonitor/description_fr.properties
Comment (jgreffe) [reply core/src/main/resources/jenkins/monitor/OperatingSystemEndOfLifeAdminMonitor/description_fr.properties:1]: Not sure here why surrounding with spaces?

Comment (lemeurherve) [reply core/src/main/resources/jenkins/monitor/OperatingSystemEndOfLifeAdminMonitor/description_fr.properties:1]: @jgreffe every other `blurb`s have spaces around `=`

**Comment by jgreffe** [comment]
Thanks for your review @lemeurherve 👍🏼 
Added a commit with suggestions related to non fr properties here: https://github.com/jenkinsci/jenkins/pull/8595/commits/de41fd73b3b417851097d07afa4503a1118d5785

File: core/src/main/resources/jenkins/monitor/OperatingSystemEndOfLifeAdminMonitor/description_fr.properties
Comment (jgreffe) [reply core/src/main/resources/jenkins/monitor/OperatingSystemEndOfLifeAdminMonitor/description_fr.properties:1]: We see some examples where it's not the case:
https://github.com/jenkinsci/jenkins/blob/ca646b3742e6921d332078d0b287fd359970d7e9/core/src/main/resources/hudson/cli/CLIAction/index.properties#L2
https://github.com/jenkinsci/jenkins/blob/c20edce9d684e4c5532a565d91eaa40f9fd9db36/core/src/main/resources/hudson/model/Computer/setOfflineCause.properties#L2

But let's do commit the suggestion.

**Review by lemeurherve — APPROVED** [review]
Great job 🙌 

Ready for merge IMO

File: core/src/main/resources/jenkins/monitor/OperatingSystemEndOfLifeAdminMonitor/description_fr.properties
Comment (jgreffe) [reply core/src/main/resources/jenkins/monitor/OperatingSystemEndOfLifeAdminMonitor/description_fr.properties]: This means I could later file a PR with all the surrounding spaces removed from the properties, including `blurb`?

**Comment by jgreffe** [comment]
> Great job 🙌
> 
> Ready for merge IMO

Team work, thanks for your review & suggestions! ❤️

File: core/src/main/resources/jenkins/monitor/OperatingSystemEndOfLifeAdminMonitor/description_fr.properties
Comment (lemeurherve) [reply core/src/main/resources/jenkins/monitor/OperatingSystemEndOfLifeAdminMonitor/description_fr.properties]: I don't know, I would ask someone more familiar with these properties and how they're consumed in Jenkins.

@jenkinsci/core-pr-reviewers WDYT?

**Comment by timja** [comment]
>  slash label localization

comma separated I think for multiple

File: core/src/main/resources/jenkins/agents/CloudSet/index.properties
Comment (timja) [inline core/src/main/resources/jenkins/agents/CloudSet/index.properties:3]: this isn't right, are is a [linking verb](https://www.yourdictionary.com/articles/is-are-sentences) and is plural.
while implementation is singular.

the existing text is bit developer specific really and I would say not user facing text, maybe this would be more appropriate (it would depend on what it looks like in place and any surrounding text I haven't reviewed that):

```suggestion
noCloudPlugin=There is no plugin installed that supports clouds.
```

File: core/src/main/resources/jenkins/security/UpdateSiteWarningsConfiguration/config.properties
Comment (timja) [inline core/src/main/resources/jenkins/security/UpdateSiteWarningsConfiguration/config.properties:27]: there may be one or more update sites, Jenkins on its own has only 1 but there's a plugin that allows you to add more and I think CloudBees may use multiple,?
```suggestion
  configuration, provided by the update site(s). If you do, they won’t be shown by \
```

File: core/src/main/resources/jenkins/agents/CloudSet/index.properties
Comment (lemeurherve) [reply core/src/main/resources/jenkins/agents/CloudSet/index.properties:3]: Much better indeed.

File: core/src/main/resources/jenkins/agents/CloudSet/index_fr.properties
Comment (jgreffe) [inline core/src/main/resources/jenkins/agents/CloudSet/index_fr.properties:3]:
```suggestion
noCloudPlugin=Aucun plugin installé supportant les clouds.
```

File: core/src/main/resources/jenkins/security/UpdateSiteWarningsConfiguration/config_fr.properties
Comment (jgreffe) [inline core/src/main/resources/jenkins/security/UpdateSiteWarningsConfiguration/config_fr.properties:26]:
```suggestion
blurb = Cette section vous permet de supprimer les avertissements fournis par le(s) site(s) de mise à jour, applicables à votre configuration Jenkins. \
```

**Review by lemeurherve — APPROVED** [review]

**Review by timja — APPROVED** [review]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
